wallets/endpoint.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package wallets
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"storj.io/storjscan/tokens"
+)
+
+// ErrEndpoint is the wallets endpoint error class.
+var ErrEndpoint = errs.Class("Wallets Endpoint")
+
+// Endpoint for interacting with the Wallets service.
+//
+// architecture: Endpoint
+type Endpoint struct {
+	log     *zap.Logger
+	service Wallets
+}
+
+// NewEndpoint creates new wallets endpoint instance.
+func NewEndpoint(log *zap.Logger, wallets Wallets) *Endpoint {
+	return &Endpoint{
+		log:     log,
+		service: wallets,
+	}
+}
+
+// Register registers endpoint methods on API server subroute.
+func (endpoint *Endpoint) Register(router *mux.Router) {
+	router.HandleFunc("/wallets", endpoint.GetNewDepositAddress).Methods(http.MethodGet)
+	router.HandleFunc("/wallets/count", endpoint.GetCountDepositAddresses).Methods(http.MethodGet)
```

Comment:
Can be more simple to use just one `/wallets/count` which returns with all the numbers (`{"claimed":12,"unclaimed":23}`). Or just `/wallets/stat` if we will put more information eventually (like how many wallets received funds...)

Reply:
good idea